src/components/organisms/EntityDetails/EntityDetailsContent/Settings/SettingsTests/Nodes/AddNode.tsx

```diff
@@ -0,0 +1,35 @@
+import {Dropdown} from 'antd';
+
+import {IntersectionContainer} from '../SettingsTests.styled';
+
+type AddNodeProps = {
+  data: {
+    stepLength: number;
+    showTestModal: (group: number | string) => void;
+    showDelayModal: (group: number | string) => void;
+    group: number | string;
+  };
+};
+
+const AddNode: React.FC<AddNodeProps> = props => {
+  const {data} = props;
+
+  return (
+    <>
+      <Dropdown
```

Comment:
Dropdown could be extracted, as it's exactly the same for both IntersectionNode and AddNode